app/views/organisations/_corporate_information.html.erb

```diff
@@ -23,4 +23,10 @@
     } %>
     <%= render "components/topic-list", @show.corporate_information[:job_links] %>
   <% end %>
+
+  <% if @organisation.secondary_corporate_information %>
+    <p class="organisation__secondary-corporate-information">
```

Comment:
Might be worth using the govspeak component to render this, would save having the extra class.

Reply:
This seems to make the text quite a lot larger than we want:

(govspeak on the left; current PR version on the right)
<img width="891" alt="screen shot 2018-06-22 at 14 57 39" src="https://user-images.githubusercontent.com/29889908/41780442-a55bec1c-762c-11e8-920e-2bd8bfddbbc1.png">

I'm not very familiar with using govspeak to render things, the docs don't suggest there's any way to adjust this?

Reply:
Oh, okay, never mind then.